Vagrantfile

```diff
@@ -851,29 +851,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # located in the www/ directory and in config/config.yml.
   #
 
-  if Vagrant.has_plugin?('vagrant-goodhosts')
+  if config.vagrant.plugins.include?('vagrant-goodhosts')
     config.goodhosts.aliases = vvv_config['hosts']
     config.goodhosts.remove_on_suspend = true
-  elsif Vagrant.has_plugin?('vagrant-hostsmanager')
+  elsif config.vagrant.plugins.include?('vagrant-hostsmanager')
     config.hostmanager.aliases = vvv_config['hosts']
     config.hostmanager.enabled = true
     config.hostmanager.manage_host = true
     config.hostmanager.manage_guest = true
     config.hostmanager.ignore_private_ip = false
     config.hostmanager.include_offline = true
-  elsif Vagrant.has_plugin?('vagrant-hostsupdater')
+  elsif config.vagrant.plugins.include?('vagrant-hostsupdater')
     # Pass the found host names to the hostsupdater plugin so it can perform magic.
     config.hostsupdater.aliases = vvv_config['hosts']
     config.hostsupdater.remove_on_suspend = true
-  else
-    show_check = true if %w[up halt resume suspend status provision reload].include? ARGV[0]
-    if show_check
-      puts ""
-      puts " X ! There is no hosts file vagrant plugin installed!"
-      puts " X You need the vagrant-goodhosts plugin (or HostManager/ HostsUpdater ) for domains to work in the browser"
-      puts " X Run 'vagrant plugin install --local' to fix this."
-      puts ""
-    end
```

Comment:
we do want this check else users may forget to install at least one of the plugins and then get confused why their sites don't load

Reply:
Yeah I can revert this, but that check is done by Vagrant as `config.vagrant.plugins` is set to `['vagrant-goodhosts']` is always executed as the checks around line 376 don't work as expected.

Perhaps a more comprehensive set of changes is needed?

Reply:
hmm we can restore it at a later date then

Reply:
I had restored this locally already, so here it is. :)